console/workspaces/pages/eval/src/subComponents/ScoreBreakdownCard.tsx

```
@@ -0,0 +1,177 @@
+const ScoreBreakdownCard: React.FC<ScoreBreakdownCardProps> = ({
+  level,
+  data,
+  isLoading,
+}) => {
+  const title = CARD_TITLES[level as "agent" | "llm"];
+
+  // Collect unique evaluator names across all groups
+  const evaluatorNames = useMemo(() => {
+    if (!data?.groups) return [];
+    const nameSet = new Set<string>();
+    data.groups.forEach((g) =>
```

Biome (2.4.4)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)